logging/google/cloud/logging/_http.py

```diff
@@ -293,7 +293,7 @@ def sink_update(self, project, sink_name, filter_, destination):
             'filter': filter_,
             'destination': destination,
         }
-        self.api_request(method='PUT', path=target, data=data)
+        return self.api_request(method='PUT', path=target, data=data)
```

Comment:
If you're gonna do it (I'm -1, personally), you need to add `:rtype:` and `:returns:` to the docstring.